pkg/logs/config/config_keys.go

```diff
@@ -15,18 +15,24 @@ import (
 
 // LogsConfigKeys stores logs configuration keys stored in YAML configuration files
 type LogsConfigKeys struct {
-	prefix string
-	config coreConfig.Config
+	prefix       string
+	vectorPrefix string
+	config       coreConfig.Config
 }
 
 // defaultLogsConfigKeys defines the default YAML keys used to retrieve logs configuration
 func defaultLogsConfigKeys() *LogsConfigKeys {
-	return NewLogsConfigKeys("logs_config.", coreConfig.Datadog)
+	return NewLogsConfigKeysWithVector("logs_config.", "logs.", coreConfig.Datadog)
```

Comment:
What is the significance of the `logs.` prefix and how it relates to vector? I don't see a case where `vectorPrefix` would be anything else since `NewLogsConfigKeysWithVector` is not called outside of `defaultLogsConfigKeys`. 

Do we plan to use `NewLogsConfigKeysWithVector` in the future in other places?

Reply:
You're right it seems unlikely to be useful in the future, I will remote it.